webapp/src/js/services/enketo.js

```diff
@@ -297,6 +313,7 @@ angular.module('inboxServices').service('Enketo',
         .off('.pagemode')
         .on('click.pagemode', function() {
           $window.history.back();
+          setupNavButtons(form, $wrapper, form.pages._getCurrentIndex() - 1);
```

Comment:
Is there any other way to access the current page number without using an internal function? This could break without warning...

Reply:
This goes for `$activePages` too...

Reply:
There's no other nice way of doing this. $activePages is exposed in later versions of enketo-core so we can at least fix that when we upgrade.